services/k8s/apps/daemonsets.yml

```diff
@@ -0,0 +1,68 @@
+name: DaemonSets
+description: >-
+  Control Kubernetes DaemonSets objects.
+scope: CRITICAL
+notes: >-
+  DaemonSets manage pods, similar to ReplicaSets and StatefulSets. A DaemonSet ensures
+  there is the desired number of pods running on each node. If DaemonSets are allowed to connect
+  to the public internet, it may open up the door for arbitrary code execution for an attacker.
+  See notes on `container/deployments` for mitigations.
+privileges:
+  create:
+    risks:
+      - impact:spend
+      - impact:hijack
+      - escalation:lateral
+      - escalation:network
+    notes: >-
+      Creation of DaemonSets allows running an image inside the Kubernetes cluster. This may allow arbitrary code
+      execution, if the cluster has access to the public internet. The code will execute with service account privileges,
+      leading to new permissions that may allow access to other GCP services. 
```

Comment:
GCP reference here. Not sure how k8s service accounts work, but maybe just replace with something about how it could lead to privilege escalation generally?